indico/modules/rb/client/js/common/bookings/actions.js

```diff
@@ -91,3 +99,24 @@ export function changeBookingOccurrenceState(id, date, action, params = {}) {
     BOOKING_OCCURRENCE_STATE_CHANGE_ERROR
   );
 }
+
+export function linkBookingOccurrence(bookingId, date, eventId, onSuccess, params = {}) {
+  return ajaxAction(
+    () =>
+      indicoAxios.post(
+        bookingOccurrenceLinkActionsURL({booking_id: bookingId, date, event_id: eventId}),
+        params
+      ),
+    BOOKING_OCCURRENCE_LINK_CHANGE_REQUEST,
+    [
+      BOOKING_OCCURRENCE_LINK_UPDATED,
```

Comment:
after a successful linking you probably want to clear the linking mode so the "Your booking will be linked to an event" bar on top of the room booking page disappearsY(you should be able to do that on the reducer level by handling this action in the the reducer in `.../linking/reducers.js`:

```
    case linkingActions.CLEAR_OBJECT:
    case bookRoomActions.CREATE_BOOKING_SUCCESS:
      return initialState;
```

Reply:
I definitely would not do that. I think it's a very common scenario to link bookings to more than one booking occurrence.

Reply:
Ah, good point. Didn't think of that usecase. I agree that it does not make much sense to exit linking mode in that case.